Gemfile

```diff
@@ -10,7 +10,6 @@ gem 'govspeak'
 gem 'govuk_design_system_formbuilder'
 gem 'lograge'
 gem 'logstash-event'
-gem 'multipart-post', '2.4.1'
```

Comment:
This is an old gem pin that has now been completed bypassed by an upgrade of faraday's major version in the UKTT